### PR TITLE
Slhc26 me0 segment time pr

### PR DESCRIPTION
**Pull request by pietverwilligen**
Pull request containing updates for GEM Reco for SLHC:
- hit clustering in time window in ME0Segment
- neutron background in all BX of ME0

### CODE REVIEW

**Comment by mark-grimes** [comment]
please test

**Comment by pietverwilligen** [comment]
I resolved the merge conflict. can you try again?
Thanks
Piet

**Comment by mark-grimes** [comment]
please test

**Comment by pietverwilligen** [comment]
i am looking into the error reported by cmsbuild ... I had tested Friday in CMSSW_6_2_SLHCDEV_X_2015-11-12-2300 and there was no problem. I will test again and correct in CMSSW_6_2_SLHCDEV_X_2015-11-15-1100 and let you know.

File: RecoMuon/MuonIdentification/test/ME0MuonAnalyzer.cc [hunk 1]
```diff
@@ -217,98 +230,154 @@ ME0MuonAnalyzer::ME0MuonAnalyzer(const edm::ParameterSet& iConfig)
 }
 
 
+  std::cout<<"At start of begin run"<<std::endl;  
 
-//void ME0MuonAnalyzer::beginJob(const edm::EventSetup& iSetup)
-void ME0MuonAnalyzer::beginJob()
```
Comment (mark-grimes) [inline RecoMuon/MuonIdentification/test/ME0MuonAnalyzer.cc]: You've taken out the function specifier, so everything after these lines the compiler thinks are supposed to be global statements.  I think that's the source of all the compiler errors.

Also, cout should be LogDebug or similar.

**Comment by pietverwilligen** [comment]
sorry still working on it ...

**Comment by pietverwilligen** [comment]
sorry, i have to close this one and open another one ...

**Pull request closed without merging**